iio-widgets/src/datastrategy/channelattrdatastrategy.cpp

```diff
@@ -0,0 +1,61 @@
+#include "datastrategy/channelattrdatastrategy.h"
+#include <QFile>
+
+#define BUFFER_SIZE 256
```

Comment:
Buffer size attribute should be a lot bigger than 256 - we could try 16k. Apply this to all iio string buffer sizes